database/engine/metadata_log/metalogpluginsd.c

```diff
@@ -206,6 +263,33 @@ PARSER_RC metalog_pluginsd_tombstone_action(void *user, uuid_t *uuid)
     return PARSER_RC_OK;
 }
 
+PARSER_RC metalog_pluginsd_host(char **words, void *user, PLUGINSD_ACTION  *plugins_action)
+{
+    char *machine_guid = words[1];
+    char *hostname = words[2];
+    char *registry_hostname = words[3];
+    char *update_every_s = words[4];
+    char *os = words[5];
+    char *timezone = words[6];
+    char *tags = words[7];
+
+    int update_every = 1;
+    if (likely(update_every_s && *update_every_s))
+        update_every = str2i(update_every_s);
+    if (unlikely(!update_every))
+        update_every = 1;
+
+    info("HOST PARSED: guid=%s, hostname=%s, reg_host=%s, update=%d, os=%s, timezone=%s, tags=%s",
```

Comment:
Since the language parsing is supposed to be common this function should be in `pluginsd_parser.c` like the other keywords of metadata log:
https://github.com/netdata/netdata/blob/42aa54eaf6a64348f23cc0c9ed0a071b79bc9cad/collectors/plugins.d/pluginsd_parser.c#L608

```suggestion
    debug(D_PLUGINSD, "HOST PARSED: guid=%s, hostname=%s, reg_host=%s, update=%d, os=%s, timezone=%s, tags=%s",
```